Source/ABLASTR/ABLASTR.H

```diff
@@ -0,0 +1,179 @@
+/* Copyright 2019 Axel Huebl, Andrew Myers
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef ABLASTR_H_
```

Comment:
Some thoughts:
- maybe let us name the file `DepositCharge.H`
- use lowercase in `namespace ablastr`